### DISCUSSION TITLE
Use of $img helper in composition API

### DISCUSSION THREAD

**Discussion by CatalinGheorghiu** [nuxt/image#575] [Q&A]
Hello

Thanks for this awesome work!

I there a way to access the `$img` utility inside composition API ( script setup )? The page on the documentation pointing to this helper seems broken (https://v1.image.nuxtjs.org/api/$img)

Thanks

**Comment by AndersNielsen85** [comment]
@CatalinGheorghiu have at look at this one, it will add it :)
https://github.com/nuxt/image/pull/571

I managed to make it work for now with a composable.
```
// @TODO remove when merged  https://github.com/nuxt/image/pull/571
import type { $Img } from '@nuxt/image-edge'
import { createImage } from '#image'
// @ts-ignore
import { imageOptions } from '#build/image-options'

export const useImage = () => {
  const img = createImage(imageOptions)

  return {
    generate: (...params: Parameters<$Img>) => img(...params),
    getImage: img.getImage,
    getMeta: img.getMeta,
    getSizes: img.getSizes,
    options: img.options
  }
}
```

Then use
`
const { generate } = useImage();`
